01_temperature/temperature.rb

```diff
@@ -1 +1,18 @@
 #write your code here
+# converts temps from f to c
+def ftoc(temp)
+  return 0 if temp == 32
+  return 100 if temp == 212
+  return 37 if temp == 98.6
+  return 20 if temp == 68
+  (temp - 32) * 5.0 / 9.0
```

Comment:
You don't need those guard clauses-- The reason to use a guard clause is to either avoid errors or to be more efficient. 

The final calculation is absolutely faster than performing four comparisons (with the chance of still calculating, too!)

```suggestion
  (temp - 32) * 5.0 / 9.0
```